### PR TITLE
Add toggle for windowed mode in Display Options.

### PR DESCRIPTION
**Pull request by rohit-n**
Taken from https://github.com/haekb/nolf2-modernizer/commit/6d8c6bd127a156745e6514cb88fcd1f8b5fe3258

### CODE REVIEW

File: NOLF2/ClientShellDLL/ClientShellShared/GameClientShell.cpp [hunk 1]
```diff
@@ -1167,7 +1183,7 @@ uint32 CGameClientShell::OnEngineInitialized(RMode *pMode, LTGUID *pAppGuid)
 		
 		// Use the settings from the current profile to determine what the game will be...
 
-		CUserProfile *pProfile = g_pProfileMgr->GetCurrentProfile( );
+		pProfile = g_pProfileMgr->GetCurrentProfile( );
```
Comment (Katana-Steel) [inline NOLF2/ClientShellDLL/ClientShellShared/GameClientShell.cpp]: does pProfile change between 1123 and here?
if not how about dropping this line?

File: NOLF2/ClientShellDLL/ClientShellShared/GameClientShell.cpp [hunk 2]
```diff
@@ -873,6 +873,8 @@ void CGameClientShell::CSPrint(const char* msg, ...)
 
 uint32 CGameClientShell::OnEngineInitialized(RMode *pMode, LTGUID *pAppGuid)
 {
+	SDL_Window* window;
```
Comment (Katana-Steel) [inline NOLF2/ClientShellDLL/ClientShellShared/GameClientShell.cpp]: move this to about or just before 1133

File: NOLF2/ClientShellDLL/ClientShellShared/GameClientShell.cpp [hunk 3]
```diff
@@ -873,6 +873,8 @@ void CGameClientShell::CSPrint(const char* msg, ...)
...
+	CUserProfile* pProfile;
```
Comment (Katana-Steel) [inline NOLF2/ClientShellDLL/ClientShellShared/GameClientShell.cpp]: just declare and initialize this on 1123, there's no need for it at the start of the function

File: NOLF2/ClientShellDLL/ClientShellShared/ProfileMgr.cpp [hunk 4]
```diff
@@ -1066,7 +1067,8 @@ void CUserProfile::ImplementMusicVolume()
 
 void CUserProfile::ApplyDisplay()
 {
-
+	SDL_Window* window;
```
Comment (Katana-Steel) [inline NOLF2/ClientShellDLL/ClientShellShared/ProfileMgr.cpp]: Again this can be moved to line 1120, window is not used anywhere else in the method

**Review by Katana-Steel — CHANGES_REQUESTED** [review]

**Comment by rohit-n** [comment]
Changes made.

**Pull request merged**